### PR TITLE
Ks 2021 06 post idea

### PR DESCRIPTION
**Pull request by Rineee**
Only posts name and description of ideas.

### CODE REVIEW

**Review by fuzzylogic2000 — COMMENTED** [review]
Reads well to me, but in the app, I still can't see any projects, so can't currently test it.

File: containers/Ideas/ExplorePage.js [hunk 1]
```diff
@@ -29,7 +21,10 @@ export const ExplorePage = (props) => {
   );
 
   useEffect(() => {
-    fetchProjects();
+    const projectsListener = props.navigation.addListener('focus', () => {
```
Comment (khamui) [inline containers/Ideas/ExplorePage.js:24]: nice! i guess this is v5.x?

Comment (khamui) [reply containers/Ideas/ExplorePage.js:24]: ah yes the commit message says it `...make listeners work with new navigation` 💁🏾‍♂️

File: containers/Auth/LoginScreen.js [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
-      Alert.alert('An error occured', error[0], [{ text: 'Ok' }]);
+      Alert.alert('An error occured', error, [{ text: 'Ok' }]);
     }
   }, [error]);
 
   const handleLogin = (values) => {
     API.postLogin(values).then((response) => {
-      if (!response.token) {
-        setError(response.non_field_errors);
+      if (!(response.statusCode==200)) {
```
Comment (khamui) [inline containers/Auth/LoginScreen.js:25]: maybe more readable `response.statusCode !== 200`?

**Review by khamui — APPROVED** [review]
Really nice. Very readable indeed, i could test it and it works fine!!
@Rineee one comment, if no need to change i can merge. @fuzzylogic2000 no further objections/findings?

**Comment by goapunk** [comment]
Looks goods to me. One think which is not just specific to this MR: The back button is white and therefore is not visible if the background image is white as well (e.g. in the `one more app project`).

**Comment by Rineee** [comment]
> Looks goods to me. One thing which is not just specific to this MR: The back button is white and therefore is not visible if the background image is white as well (e.g. in the `one more app project`).

Yes, I also saw that. Do you want to open an issue for it?

**Comment by khamui** [comment]
yeah!

**Pull request merged**